library/dnet.c

```diff
@@ -46,6 +46,29 @@
 
 #include "logger.hpp"
 
+int dnet_remove_local_new(struct dnet_backend_io *backend, struct dnet_node *n, struct dnet_id *id,
+			  void *packet, size_t packet_size)
```

Comment:
1. Current architecture of *_NEW commands is totally broken: request parsing occurs on backend level instead of transport level. In ideal world we should just replace the flag on the request that is passed by calling code to the cache subsystem.
2. dnet.c cant't invoke c++ template function.